.github/dependabot.yml

```
@@ -9,3 +9,7 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+  - package-ecosystem: "yarn"
```

```suggestion
  - package-ecosystem: "npm"
```

The error we got was:
> The property '#/updates/1/package-ecosystem' value "yarn" did not match one of the following values: npm, bundler, composer, maven, mix, cargo, gradle, nuget, gomod, docker, elm, gitsubmodule, github-actions, pip, terraform, pub